R-packages/evalcast/vignettes/intro-evalcast.Rmd

```diff
@@ -44,87 +48,142 @@ library(evalcast)
 library(covidcast)
 library(lubridate)
 
-forecast_dates <- get_forecast_dates("CMU-TimeSeries")
-forecast_dates_july <- forecast_dates[forecast_dates >= "2020-07-01" & forecast_dates <= "2020-07-31"]
+forecast_dates <- get_covidhub_forecast_dates("CMU-TimeSeries")
+forecast_dates_july <- forecast_dates[forecast_dates >= "2020-07-01" & 
+                                        forecast_dates <= "2020-07-31"]
 
 # Retrieve past predictions from CovidHub
-predictions_cards_yyg <- get_covidhub_predictions("YYG-ParamSearch", forecast_dates_july)
-predictions_cards_cmu <- get_covidhub_predictions("CMU-TimeSeries", forecast_dates_july)
-
+predictions_cards_ens <- get_covidhub_predictions(
+  "COVIDhub-ensemble", forecast_dates_july, ahead = 3, 
+  signal = "deaths_incidence_num") %>%
+  filter(nchar(geo_value)==2) # remove counties
+predictions_cards_cmu <- get_covidhub_predictions(
+  "CMU-TimeSeries", forecast_dates_july,  ahead = 3, 
+  signal = "deaths_incidence_num")
 # Make new predictions using baseline_forecaster
```

Comment:
This isn't a change you made, but it might help to split the `get_covidhub_predictions` calls from the `get_predictions` call, it's a bit confusing until you realize that they don't affect each other.

I think even adding ellipsis to the comments could help a lot:
```
# Retrieve past predictions from Covidhub...

# ... or make new predictions using baseline_forecaster.
```